modules/network/sepolia.ts

```diff
@@ -22,7 +22,7 @@ export const sepoliaNetworkConfig: NetworkConfig = {
         new YbTokensAprService(sepoliaNetworkData.ybAprConfig, sepoliaNetworkData.chain.prismaId),
         new BoostedPoolAprService(),
         new SwapFeeAprService(),
-        new GaugeAprService(tokenService, [sepoliaNetworkData.bal!.address]),
```

Comment:
should merge v3-canary into this branch and change this for the sonic config as well